tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml

```diff
@@ -104,6 +104,8 @@ files:
       interface-name=eth0
       master={{ IRONIC_ENDPOINT_BRIDGE }}
       slave-type=bridge
+      autoconnect=yes
+      autoconnect-priority=999
```

Comment:
999 means highest priority? How did you come up with that number ?

Reply:
not the highest number. It is int32. I used an example from [here](https://www.redhat.com/sysadmin/becoming-friends-networkmanager). Basically any number above 0 works in our case.